library/psa_crypto.c

```diff
@@ -6052,11 +6053,11 @@ static psa_status_t psa_generate_key_internal(
             return( status );
 
         slot->attr.bits = (psa_key_bits_t) bits;
-#if defined(MBEDTLS_DES_C)
+#if defined(PSA_WANT_KEY_TYPE_DES)
```

Comment:
I'll change these.

Reply:
I can change these, but I'm still not convinced that it is correct. They are conditionalizing the use of PSA_KEY_TYPE_DES, which seems to me to match the WANT macro. In either case, it wouldn't matter since the code is only in SW.